contracts/0.0.1/group-slim.js

```diff
@@ -1464,7 +1464,7 @@ ${this.getErrorInfo()}`;
           }
         },
         process({ data, meta }, { state }) {
-          import_common2.Vue.delete(state.chatRooms, data.chatRoomID);
+          import_common2.Vue.delete(state.chatRooms[data.chatRoomID]);
```

Comment:
Looks like this got reverted in the recent commit. Did you undo the `grunt pin:0.0.1` that you did earlier? Remember you need to delete the old `contracts/0.0.1` folder and run that command each time you update this PR.

Reply:
I removed `0.0.1` folder and run `grunt pin:0.0.1`. Also I tried it again just before but it works that way.
Weird...

Reply:
Should I run `grunt dev` and then `grunt pin:0.0.1`?

Reply:
I did this way and it is resolved...